docs/recipes/typescript.md

```diff
@@ -0,0 +1,56 @@
+# TypeScript
+
+AVA comes bundled with a TypeScript definition file. This allows developers to leverage TypeScript for writing tests.
+
+## Setup
+
+First install the TypeScript compiler [tsc](https://github.com/Microsoft/TypeScript).
+
+```
+$ npm install tsc --save-dev
+```
+
+Create a [`tsconfig.json`](https://github.com/Microsoft/TypeScript/wiki/tsconfig.json) file. This file specifies the compiler options required to compile the project or the test file.
+
+```json
+{
+	"compilerOptions": {
+		"module": "commonjs",
+		"target": "es2015"
```

Comment:
Can you also add `outDir`? That will compile the sources to a different directory. If you place all files in the same directory, you can easily confuse JS fileswith TS files.